package.json

```diff
@@ -2,10 +2,12 @@
   "name": "pooper",
   "author": "gamasak",
   "private": true,
+  "main": "pooper",
   "scripts": {
     "start": "node index.js"
   },
   "dependencies": {
     "express": "^4.16.2"
```

Comment:
please specify specific version

Reply:
hehe. What version do you think is suitable?